src/Microsoft.IdentityModel.Tokens/CryptoProviderFactory.cs

```diff
@@ -34,13 +34,13 @@ namespace Microsoft.IdentityModel.Tokens
     /// Creates <see cref="SignatureProvider"/>s by specifying a <see cref="SecurityKey"/> and algorithm.
     /// <para>Supports both <see cref="AsymmetricSecurityKey"/> and <see cref="SymmetricSecurityKey"/>.</para>
     /// </summary>
-    public class SignatureProviderFactory
+    public class CryptoProviderFactory
```

Comment:
@PinpointTownes thanks. We still have some final smoothing out here. I am not convinced we havew a good extensibility cross plat story. If you can spare some cycles and have a look once this PR is in, we would appreciate it.